docs/provider/delinea.md

```diff
@@ -0,0 +1,45 @@
+## Delinea DevOps Secrets Vault
+
+External Secrets Operator integrates with [Delinea DevOps Secrets Vault](https://docs.delinea.com/online-help/products/devops-secrets-vault/current).
+
+### Creating a SecretStore
+
+You need client ID, client secret and tenant to authenticate with DSV.
```

Comment:
Could you please describe how to acquire these credentials? I'm struggling to set this up. Maybe it's the trial account that has that feature disabled? :sweat_smile:

Reply:
Oh sorry, I also struggled a bit with this in the beginning. For the implementation I only used a trial account though, so I am certain it works with that.

Did you follow steps 1-3 of https://docs.delinea.com/dsv/current/quickstart/obtaindsv? The CLI should be initialised with your "Thycotic One" user I believe (the user you used to sign up for the trial).

Then, you can create a client, which is described at https://docs.delinea.com/dsv/current/usage/cli-ref/client.md. The output of that command should give you the required ID and secret.

Does that help or are you stuck somewhere along the steps?

Reply:
If it’s gets confusing I’ve built a few scripts to help get a client credential setup for a few test scenarios such as [DSV github action](https://github.com/DelineaXPM/dsv-github-action#prerequisites).  Let me know if I can tweak to help reduce any review friction.   I did this to help jump start test usage and maybe it will cut down the platform specific knowledge required.

Reply:
Makes sense, I added some notes in 15de5ad9f60db0d6ede0f0f5b3143a0fa5976ab2, let me know if that helps.